stage-1-proposals.md

```diff
@@ -53,7 +53,7 @@ Proposals follow [this process document](https://tc39.es/process-document/).
 | [OOM Fails Fast][oom]                                                                        | Mark Miller                                           | Mark Miller                                            | <sub>[December&nbsp;2019][oom-notes]</sub>                        |
 | [Array filtering][array-filtering]                                                           | Justin Ridgewell                                      | Justin Ridgewell                                       | <sub>[February&nbsp;2020][array-filtering-notes]</sub>            |
 | [Async initialization][async-init]                                                           | Bradley Farias                                        | Bradley Farias                                         | <sub>[February&nbsp;2020][async-init-notes]</sub>                 |
-| [Decimal][decimal]                                                                           | Daniel Ehrenberg<br />Andrew Paprocki                 | Philip Chimento<br />Andrew Paprocki<br />Jesse Alama  | <sub>[December&nbsp;2021][decimal-notes]</sub>                    |
+| [Decimal][decimal]                                                                           | Daniel Ehrenberg<br />Andrew Paprocki                 | Andrew Paprocki<br />Jesse Alama<br />Jirka Marsik  | <sub>[September&nbsp;2023][decimal-stage-1-update-and-discussion]</sub>                    |
```

Comment:
the link should be `decimal-notes`, and `decimal-stage-1-update-and-discussion` doesn't link anywhere (footnote links are at the bottom of the document). where should this link to?